deepchem/models/torch_models/modular.py

```diff
@@ -62,7 +62,7 @@ class ModularTorchModel(TorchModel):
     ...     return (torch.nn.functional.mse_loss(pretrain_model(inputs), labels[0]) * weights[0]).mean()
     >>> pretrain_modular_model.loss_func = example_pt_loss_func
     >>> pt_loss = pretrain_modular_model.fit(dataset_pt, nb_epoch=1)
-    >>> modular_model.load_pretrained_components(pretrain_modular_model, components=['linear'])
+    >>> modular_model.load_from_pretrained(pretrain_modular_model, components=['linear'])
```

Comment:
In a separate PR, I think we can move this usage example into a jupyter notebook tutorial for how to use `ModularTorchModel` to make it more clear. For now, I think it's good to have a basic example of how to use ModularTorchModel here

Reply:
Also, just realized the `ModularTorchModel` is not rendering in the docs due to a docs build breakage (https://github.com/deepchem/deepchem/issues/3306)